src/commands/createWebApp/setPrePromptDefaults.ts

```diff
@@ -22,17 +21,13 @@ export async function setPrePromptDefaults(wizardContext: IWebAppWizardContext &
         const fsPath: string = workspaceForRecommendation.uri.fsPath;
 
         if (await fse.pathExists(path.join(fsPath, 'package.json'))) {
-            wizardContext.recommendedSiteRuntime = [LinuxRuntimes.node];
-
+            wizardContext.recommendedSiteRuntime = ['node'];
```

Comment:
Did you stop using `LinuxRuntimes` because these runtimes aren't necessarily for linux?

Reply:
Mostly because I wanted to use `WebAppStackValue` instead (from `WebAppStackModel.ts`). That type is copied from the source repo for the stacks API, so it'll be easier for us to stay in-sync. Keep in mind, it might look like a simple string, but it's not - we still get standard errors for things like misspelling, etc.:

<img width="756" alt="Screen Shot 2020-11-05 at 1 26 04 PM" src="https://user-images.githubusercontent.com/11282622/98298248-8ac39500-1f6a-11eb-80c8-349847a99c83.png">